server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -50,6 +68,11 @@ case class NettyConfig(
 
   def socketBacklog(s: Int): NettyConfig = copy(socketBacklog = s)
 
+  def withRequestTimeout(r: FiniteDuration): NettyConfig = copy(requestTimeout = Some(r))
```

Comment:
other methods e.g. `port` or `maxCointentLength` don't have the `with` prefix - maybe we should keep consistency here as well?